build.sh

```diff
@@ -261,4 +268,62 @@ build_ios_dylib() {
   printf "%s\n" "Successfully built Leanplum-SDK (iOS) Framework.\n"
 }
 
+#######################################
+# Builds the dynamic library tvOS target.
+# Globals:
+#   None
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+build_tvos_dylib() {
+  echo "Starting build for Leanplum-SDK (tvOS)"
+
+  mkdir -p "${RELEASE_DIR}/tvos/"
+
+  run "Building Leanplum-SDK-tvOS dynamic (device/arm64) target ..." \
+    xcodebuild -configuration "${CONFIGURATION}" -target "Leanplum-iOS-SDK-source-tvOS" -sdk "${TVOS_SDK}" \
+    "$ACTION" ARCHS='arm64' RUN_CLANG_STATIC_ANALYZER=NO BUILD_DIR="${BUILD_DIR}${ARM64_DIR}" \
+    BUILD_ROOT="${BUILD_ROOT}" OTHER_CFLAGS="-fembed-bitcode" \
+    GCC_PREPROCESSOR_DEFINITIONS="PACKAGE_IDENTIFIER=${LEANPLUM_PACKAGE_IDENTIFIER}"
+  run "Building Leanplum-SDK-tvOS dynamic (simulator/x86_64) target ..." \
+    xcodebuild -configuration "${CONFIGURATION}" -target "Leanplum-iOS-SDK-source-tvOS" -sdk "${TVOS_SIM_SDK}" \
+    "$ACTION" ARCHS='x86_64' VALID_ARCHS='x86_64' RUN_CLANG_STATIC_ANALYZER=NO \
+    BUILD_DIR="${BUILD_DIR}${X8664_DIR}" BUILD_ROOT="${BUILD_ROOT}" OTHER_CFLAGS="-fembed-bitcode" \
+    GCC_PREPROCESSOR_DEFINITIONS="PACKAGE_IDENTIFIER=${LEANPLUM_PACKAGE_IDENTIFIER}"
+
+  run "Combining builds to universal fat library ..." \
+    lipo -create -output "${RELEASE_DIR}/tvos/Leanplum" \
+    "${CURRENTCONFIG_ARM64_TV_DEVICE_DIR}/Leanplum-iOS-SDK-source-tvOS/Leanplum.framework/Leanplum" \
+    "${CURRENTCONFIG_X8664_TV_SIMULATOR_DIR}/Leanplum-iOS-SDK-source-tvOS/Leanplum.framework/Leanplum"
+
+  # Copy generated framework
+  cp -r "${BUILD_DIR}$ARM64_DIR/$CONFIGURATION-appletvos/Leanplum-iOS-SDK-source-tvOS/"\
+"Leanplum.framework/" "${RELEASE_DIR}/tvos/Leanplum.framework"
+  rm -f "${RELEASE_DIR}/tvos/Leanplum.framework/Leanplum"
+  mv "${RELEASE_DIR}/tvos/Leanplum" "${RELEASE_DIR}/tvos/Leanplum.framework/"
+
+  # Delete unnecessary headers
+  mv "${RELEASE_DIR}/tvos/Leanplum.framework/Headers/Leanplum.h" \
+    "${RELEASE_DIR}/tvos/Leanplum.framework/"
+  mv "${RELEASE_DIR}/tvos/Leanplum.framework/Headers/LPInbox.h" \
+    "${RELEASE_DIR}/tvos/Leanplum.framework/"
+  rm -rf "${RELEASE_DIR}/tvos/Leanplum.framework/Headers"
+  mkdir "${RELEASE_DIR}/tvos/Leanplum.framework/Headers"
+  mv "${RELEASE_DIR}/tvos/Leanplum.framework/Leanplum.h" \
+    "${RELEASE_DIR}/tvos/Leanplum.framework/Headers/"
+  mv "${RELEASE_DIR}/tvos/Leanplum.framework/LPInbox.h" \
+    "${RELEASE_DIR}/tvos/Leanplum.framework/Headers/"
+
+  rm -rf "${RELEASE_DIR}/tvos/Leanplum.framework/_CodeSignature"
+
+  # Update modulemap with correct import, since umbrella header is not generated by cocoapods with
+  # a custom module_name set.
+  sed -i "" -e "s/Leanplum-SDK-tvOS-umbrella.h/Leanplum.h/g" \
+    "${RELEASE_DIR}/tvos/Leanplum.framework/modules/module.modulemap"
+
+  printf "%s\n" "Successfully built Leanplum-SDK (tvOS) Framework.\n"
+}
+
```

Comment:
I assume you copied over from lpm. Can you make sure to have a commit that manages this too?

Reply:
The lpm ios:build will build tvos also.